flink-connectors/flink-connector-files/src/test/java/org/apache/flink/connector/file/table/FileSystemCommitterTest.java

```diff
@@ -173,7 +206,14 @@ void testEmptyPartition() throws Exception {
 
         assertThat(emptyPartitionFile).exists();
         assertThat(emptyPartitionFile).isDirectory();
-        assertThat(emptyPartitionFile).isEmptyDirectory();
+        files =
+                FileUtils.listFilesInDirectory(
+                                Paths.get(emptyPartitionFile.getPath()),
+                                (path) ->
+                                        !path.toFile().isHidden()
+                                                && !path.toFile().getName().startsWith("_"))
```

Comment:
use 
`!path.toFile().getName().equals("_SUCCESS")`?
Also, it'll be better to make `_SUCCESS` a static constant if we use it in some test methods.

Reply:
I think this can be simplified into 

```java
assertThat(emptyPartitionFile)
                .isDirectoryNotContaining(file -> !file.getName().equals("_SUCCESS"));
```

There shouldn't be hidden files in the directory, right?

Reply:
Thanks for the suggestion